src/pages/BrowserPage.qml

```diff
@@ -40,7 +40,7 @@ Page {
     }
 
     function bringToForeground(window) {
-        if (!Qt.application.active && window) {
+        if ((webView.visibility == QtWindow.Window.Minimized) && window) {
```

Comment:
How would webView.visible work generally? Shorter and perhaps takes care of Window.Hidden?

Reply:
It's still visible when on cover but updated a bit other way round.